test_vm/src/lib.rs

```diff
@@ -424,6 +434,10 @@ impl<'bs> VM<'bs> {
         let mut a = self.get_actor(from_id).unwrap();
         let call_seq = a.call_seq_num;
         a.call_seq_num = call_seq + 1;
+        // EthAccount abstractions turns Placeholders into EthAccounts
+        if a.code == *PLACEHOLDER_ACTOR_CODE_ID {
+            a.code = *ETHACCOUNT_ACTOR_CODE_ID;
+        }
```

Comment:
Grumble VM magic voodoo privileging the EVM

Reply:
(Dropping it)

Reply:
Thanks. I infer that there's similar code in the FVM, so my grumble transitively applies there. Thanks for dropping it, but if this is how the FVM behaves then this is probably going to end up back here again, until the FVM itself doesn't have ethereum-specifics inside it.